saleor/graphql/core/types/__init__.py

```diff
@@ -1,15 +1,5 @@
 # flake8: noqa
-from .common import (
-    CountryDisplay,
-    Error,
-    File,
-    Image,
-    LanguageDisplay,
-    Permission,
-    SeoInput,
-    TaxType,
-    Weight,
-)
+from .common import *
```

Comment:
Is it expected? Wouldn't explicit declaration be better? This will also import imported dependencies from `.common` (if any exists there)

Reply:
I was explicit before and was missing a lot of imports. But I'll revert it and add all missing ones.